QuickDeployment/QuickDeploymentClientSetup.md

```diff
@@ -2,41 +2,52 @@
 
-* Setting up DNS on the client to access QDE.
-* Install the QDE SSL/TLS certificate so clients can access HTTPS components
-* Install Chocolatey and friends
+- Setting up DNS on the client to access QDE.
+- Install the QDE SSL/TLS certificate so clients can access HTTPS components
+- Install Chocolatey and friends
 
 ## DNS
-Typically in your environment, onces you've added QDE, it should be able to start talking to the QDE. In some situations, you may need to add the host name with the IP address to reach your environment.
+
+Typically in your environment, onces you've added QDE, it should be able to start talking to the QDE.
+In some situations, you may need to add the host name with the IP address to reach your environment.
 
 ## Client Installation
 
 On your client machines, you will be running the following script in an administrative context:
 
 ```powershell
-Set-ExecutionPolicy RemoteSigned -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/Import-QuickDeployCertificate.ps1')); Set-ExecutionPolicy RemoteSigned -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocoserver:8443/repository/choco-install/ClientSetup.ps1'))
+Set-ExecutionPolicy RemoteSigned -Scope Process -Force
+iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/Import-QuickDeployCertificate.ps1'))
+Set-ExecutionPolicy RemoteSigned -Scope Process -Force
+iex ((New-Object System.Net.WebClient).DownloadString('https://chocoserver:8443/repository/choco-install/ClientSetup.ps1'))
 ```
 
 What does this do?
-* Sets the execution policy for this script run to remote signed scripts. This is only in the scope of this process and not permanent.
-* Imports the SSL Certificate from the Quick Deploy Environment. **NOTE**: This is a signed script that is used to import a certificate. Due to how it works and security considerations, there are very few options allowed.
-* Switches execution policy to bypass for the internal script. This is only in the scope of this process and not permanent.
-* Calls Client setup script from the QDE environment (see below for what it does).
 
-> :warning: **WARNING**: If your clients are airgapped, you will need to find a different means to import the QDE Certificate. Please reach out to support for options.
+- Sets the execution policy for this script run to remote signed scripts.
+  This is only in the scope of this process and not permanent.
+- Imports the SSL Certificate from the Quick Deploy Environment.
+  **NOTE**: This is a signed script that is used to import a certificate.
+  Due to how it works and security considerations, there are very few options allowed.
+- Switches execution policy to bypass for the internal script.
+  This is only in the scope of this process and not permanent.
+- Calls Client setup script from the QDE environment (see below for what it does).
```

Comment:
Not sure how Markdig will respond to this.

Reply:
Assuming it's responding according to Markdown standard like Github etc., it should be OK. If not, we can avoid it.

Reply:
We can just go and see how it ends up.